ib_insync/wrapper.py

```diff
@@ -783,7 +783,11 @@ def tickString(self, reqId: int, tickType: int, value: str):
         if not ticker:
             return
         try:
-            if tickType == 47:
+            if tickType == 32:
+                ticker.bidExchange = value
+            elif tickType == 33:
+                ticker.askExchange = value
+            elif tickType == 47:
```

Comment:
Can you add a test for tickType == 84 as well, for setting ticker.lastExchange

Reply:
added in new commit